libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -290,6 +290,24 @@ public QnAMakerDialog([CallerFilePath] string sourceFilePath = "", [CallerLineNu
         [JsonProperty("rankerType")]
         public StringExpression RankerType { get; set; } = new StringExpression(RankerTypes.DefaultRankerType);
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to enable PreciseAnswer generation. 
+        /// </summary>
+        /// <value>
+        /// A value indicating user's choice between precise answer and full text in response to QnAMaker GenerateAnswer query.
+        /// </value>
+        [JsonProperty("enablePreciseAnswer")]
+        public bool EnablePreciseAnswer { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the user only wants to receive precise answer. 
+        /// </summary>
+        /// <value>
+        /// A value that indicates if user wants to receive full text along with the precise answer or not.
+        /// </value>
+        [JsonProperty("displayPreciseAnswerOnly")]
+        public bool DisplayPreciseAnswerOnly { get; set; }
```

Comment:
>bool [](start = 15, length = 4)

BooleanProperty?  Update schema file

Reply:
Updated.

---
In reply to: [437801159](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r437801159) [](ancestors = 437801159)